packages/components/psammead-radio-schedule/src/StartTime/index.jsx

```diff
@@ -19,8 +19,8 @@ const StyledClock = styled.span`
   align-items: center;
   ${({ dir }) =>
     dir === 'ltr'
-      ? `padding-right: ${GEL_SPACING_HLF}; padding-left: ${GEL_SPACING};`
-      : `padding-left: ${GEL_SPACING_HLF}; padding-right: ${GEL_SPACING};`}
+      ? `padding-right: ${GEL_SPACING_HLF};`
```

Comment:
Removing the external padding from this component, external paddings and margins (if any)should be applied from the components that import this psammead-components